FirebaseAuth/Sources/SystemService/FIRAuthStoredUserManager.h

```diff
@@ -66,9 +66,9 @@ NS_ASSUME_NONNULL_BEGIN
         we use API KEY.
     @param outError Return value for any error which occurs.
  */
-- (FIRUser *)getStoredUserForAccessGroup:(NSString *)accessGroup
-                       projectIdentifier:(NSString *)projectIdentifier
-                                   error:(NSError *_Nullable *_Nullable)outError;
+- (FIRUser *_Nullable)getStoredUserForAccessGroup:(NSString *)accessGroup
```

Comment:
nit: `- (nullable FIRUser *)...` looks more usual to me but I don't mind leaving it as is.

Reply:
Originally had it to match the above function, but I'll change them both to `nullable`.